docs/general/blockchain-security/create-wallet.md

```diff
@@ -0,0 +1,75 @@
+---
+displayed_sidebar: generalSidebar
+---
+# Security Advice for Creating a Wallet
+
+The first step into the blockchain world is creating a secure wallet. This guide provides crucial security advice for creating your wallet.
+
+## 1. Understanding the Essence of Wallets
+
+A blockchain wallet is essentially a tool for managing your private keys. The private key is a randomly generated string of numbers, while the mnemonic phrase is a human-readable version of the private key. For example:
```

Comment:
This is not accurate. Mnemonic is human-readable version of random seed and private key can be derived from it